waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -1672,4 +1674,15 @@ public CompactionResponse compact2(CompactionRequest rqst) throws TException {
     return mapping.getClient().compact2(mapping.transformInboundCompactionRequest(rqst));
   }
 
+  // Hive 2.3.7 methods
```

Comment:
From what I can tell - https://issues.apache.org/jira/browse/HIVE-21859 this was actually added in 2.3.6.